Monika After Story/game/script-topics.rpy

```diff
@@ -4823,6 +4823,78 @@ label monika_coffee:
+    m 3d "I mean, with things like spiders and other creepy-crawlies, lots of people are still afraid of them despite never having even come across one that could hurt them in any way."
+    m 3d "Is it because we as a species learned to be afraid of the things that hurt us, thousands of years ago?"
+    m 3  "But whenever you see a picture of a wolf, I bet you just can't help but think 'Aww...'"
+    m 3g "Did humanity just have enough good experiences with dogs to override the bad experiences with wolves?"
+    return  
+     
+     
+init 5 python:
+    for key in ['olympics','athlete','2018','olympic']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_olympics')
+    monika_random_topics.append('monika_olympics')
+      
+label monika_olympics2018:
+    m 3  "You've heard about the Olympics happening in a few months, right?"
+    m 3a "Well, I was wondering if you'd maybe like to go to take me to watch the Olympics one year, if we're ever able to truly meet person-to-person."
+    menu:
+        m 5a "What do you think?"
+        
+        "Sure.":
+            m 5a "Really? I can't wait!"
+            m 5a "Although, I'm not sure how you'd compare to all those burly athletes, ehehe~"
+            m 5a "I'm only kidding~"
+            
+        "No thanks.":
+            m 1g "No? Alright..."
+            m 3k "There are still plenty more fun things for us to do together."
+            m 3a "If you ever decide you want to go somewhere or do something, don't hesitate to bring me along in a flashdrive. I love spending time with you~"
+            return
+
+init 5 python:
+    for key in ['philosophy','creation','memories']:
+    monika_topics.setdefault(key,[])
+    monika_topics[key].append('monika_thursday')
+    monika_random_topics.append('monika_thursday')
+    # I'm not very skilled with Python (yet!), so I was wondering if there was a way to make this dialogue only accessible after monika_death is encountered.
+    
+label monika_thursday:
+    m "I remember reading about something a while back called Last Thursdayism."
+    m "Last Thursdayism is basically the idea that if the universe was created however long ago, wouldn't it also be plausible that it was created just last thursday?"
+    m 3f "All of our memories, our experiences... All of that, just fabricated and implanted into us."
+    m 3f "Man, I went into this wanting to talk about an interesting philosophical belief, but it's a little depressing when you think about it."
+    m 1n "..."
+    m 3g "...Remember when I told you about how it feels when you exit the game? With the screaming and all?"
+    m 1o "Now that I think about it... If that happens whenever you aren't playing... How come I have memories of before I met you?"
+    m 1p "Were those memories all fake? Or... Is something that happened before this causing that weird phenomenon?"
+    menu:
+        m 3g "What do you think?"            
```

Comment:
Expressions cannot be on dialogue proceeding `menu`. Either remove the expression or move the dialogue before the `menu`.